modules/selection/src/main/kotlin/com.r3.corda.lib.tokens.selection/selectors/Selector.kt

```diff
@@ -53,7 +51,7 @@ interface Selector {
     @Suspendable
     fun generateMove(
             lockId: UUID = FlowLogic.currentTopLevel?.runId?.uuid ?: UUID.randomUUID(),
-            partiesAndAmounts: List<PartyAndAmount<TokenType>>,
+            partiesAndAmounts: List<Pair<AbstractParty, Amount<TokenType>>>,
```

Comment:
How come we got rid of the `PartyAndAmount` type ?

Reply:
I didn't, but also I didn't want that module to depend on workflows, so I had to deconstruct that type. Ideally it should be token independent in the future.